src/elements/content-sidebar/activity-feed/utils/formatTaggedMessage.js

```diff
@@ -12,8 +14,9 @@ import UserLink from '../common/user-link';
 // mentions: ([@＠﹫]\[[0-9]+:[^\]]+])
 // urls: (?:\b)((?:(?:ht|f)tps?:\/\/)[\w\._\-]+(:\d+)?(\/[\w\-_\.~\+\/#\?&%=:\[\]@!$'\(\)\*;,]*)?)
 // NOTE: There are useless escapes in the regex below, should probably remove them when safe
-// eslint-disable-next-line
-const splitRegex = /((?:[@＠﹫]\[[0-9]+:[^\]]+])|(?:\b(?:(?:ht|f)tps?:\/\/)[\w\._\-]+(?::\d+)?(?:\/[\w\-_\.~\+\/#\?&%=:\[\]@!$'\(\)\*;,]*)?))/gim;
+/* eslint-disable no-useless-escape */
```

Comment:
i think keep the `next-line` part otherwise it disables `no-useless-escape` for the entire file
```suggestion
// eslint-disable-next-line no-useless-escape
```